### PR TITLE
Simple selenium test runner

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
I added a simple test runner to run multiple selenium tests from multiple files.

I also added a selenium for the thank-you page, which is kinda busted right now...

@alicoding r?

**Comment by alicoding** [comment]
https://www.npmjs.com/package/selenium-test-runner?

**Comment by ScottDowne** [comment]
I couldn't figure it out. Also looked like it runs tests on only one url?

**Comment by ScottDowne** [comment]
Looking at it a bit more, I think it's meant to work with the downloaded selenium and not the npm selenium-driver thingy we use.

File: tests/selenium/start.js [hunk 1]
```diff
@@ -5,64 +5,16 @@ var driver = new webdriver.Builder().
   withCapabilities(webdriver.Capabilities.firefox()).
   build();
 
+var testRunner = require('./test-runner');
+var sequentialForm = require('./sequential-form-test.js')(driver, By);
```
Comment (alicoding) [inline tests/selenium/start.js]: Why passing `By`? Why not just require it there since we don't even use it in this file?

Comment (ScottDowne) [inline tests/selenium/start.js]: EH, just trying it with one less thing to remember to require in for each test file.

Comment (ScottDowne) [inline tests/selenium/start.js]: Makes for a tighter test file.

Comment (ScottDowne) [inline tests/selenium/start.js]: I also do all requiring in this file, currently.

**Pull request merged**